.github/workflows/release.yml

```
@@ -66,13 +66,13 @@ jobs:
           # Store the hash in a file, which is uploaded as a workflow artifact.
           sha256sum $ARTIFACTS | base64 -w0 > artifacts-sha256
       - name: Upload build artifacts
-        uses: actions/upload-artifact@v4.0.0
+        uses: actions/upload-artifact@v3
```

Can we use an exact version?

```suggestion
        uses: actions/upload-artifact@v3.0.2
```

---

Done, See commit [71069d1](https://github.com/micronaut-projects/micronaut-project-template/pull/433/commits/71069d15213dc8eb2832cb4d4c85bbeee7d8a683).